ci/requirements_all.txt

```diff
@@ -3,6 +3,7 @@ pytest-cov
 pytest-xdist
 flake8
 sphinx
```

Comment:
yeah I think these should be more clear what the purpose of these are. You can add a comment (or 2) at the top of the file I think (# is ignored)